Reproducibility/src/main/java/org/tribuo/reproducibility/ReproUtil.java

```diff
@@ -299,30 +329,33 @@ private ReproUtil(ModelProvenance provenance, Model<?> originalModel) {
     }
 
     /**
-     * Return a {@link Dataset} used when a model was trained
-     * @param <T> The {@link Dataset} type
-     * @return A new {@link Dataset}
-     * @throws IllegalStateException
-     * @throws ClassNotFoundException
+     * Return a {@link Dataset} used when a model was trained.
```

Comment:
Do we want to note here that we currently only handle train/test for `TrainTestSplitter`and not any kind of cross-validation?

Reply:
Probably. The information is in the provenance, but we'd need a special case for `DatasetView` in `recoverDataset`.

I'm not sure how many models there are trained using cross validation though. While we support it the API is in need of some attention.

Reply:
Fixed.